source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -47,6 +47,20 @@ Side note 2. [If you haven't joined the Ember Community on Discord, do so today]
 
 ---
 
+## [Reason #2. Stable and More Reliable Than Ever 💪](https://emberjs.com/ember-community-survey-2019/#MS_Q401)
+
+According to the results of the latest community survey, a large proportion of the community has kept their Ember apps up to date with recent releases. Most Ember apps beyond 3.x were currently locked in at [version 3.7, 3.4 and 3.8](https://emberjs.com/ember-community-survey-2019/#MS_Q401).
+
+Seeing that a great number of Ember apps has managed to keep up with some of the latest is quite striking. It shows that Ember continues to deliver on its promise of **Stability without Stagnation**. As a framework which has been thriving in the fast-paced front-end ecosystem for over seven years, it always promoted itself as a stable and reliable solution for developers who want to make sure that the applications they're building are not only functional, but also still easy to maintain and upgrade tomorrow.
+
+A major contributing factor for the project's stability is the fine-tuned release process and the straight forward upgrade path. The release process has been refined several times in the past. Since the old ages of Ember 1.0 more than 5 years ago, a six-week release cycle had been introduced, [each one of them including beta and canary builds, as well as feature flags for not yet released, public API changes](https://blog.emberjs.com/2013/09/06/new-ember-release-process.html). A few years later, the [introduction of LTS releases](https://blog.emberjs.com/2016/02/25/announcing-embers-first-lts.html) made it even easier for developers to keep up with important updates and bug fixes, while still being able to upgrade their applications smoothly in their own time.
+
+The **success of the LTS release** is also reflected in the results of the latest survey, in which the [2.18 and the 3.4 LTS release](https://emberjs.com/ember-community-survey-2019/#MS_Q401) are among the most popular versions of Ember that today's apps depend on.
+
+Ember's success to provide stability in the maintenance process of applications also shines through when looking at the age of applications built by the survey participants: Almost a third of the applications were [more than 4 years old](https://emberjs.com/ember-community-survey-2019/#MS_Q409). This emphasises that the community is not only able to build amazing products, but that it is also **empowered to maintain** those **apps over an extraordinary long time** without a rebuild.
```

Comment:
I think the first sentence could be shortened. How about:

"Ember's success in providing stability and reliability shines also through the age of applications that the survey participants have built."?

(emphasises <- this took me a while to understand 😄)